Automattic/newspack-popups
# feat: add option for dismiss button alignment

### All Submissions:






### Changes proposed in this Pull Request:



Adds a new option/attribute for setting the alignment of the "I'm not interested" dismiss button. Looks like this:

<img width="267" alt="Screen Shot 2020-11-16 at 9 54 06 AM" src="https://user-images.githubusercontent.com/2230142/99283398-346c1700-27f2-11eb-9ec6-596f4ef68af1.png">

Default is center, so it shouldn't affect existing campaigns.

Depends on #307 because it was easier to slot this change into the reorganized sidebar options—I can rebase this PR to master if that one gets merged first.

Note that changing this setting has no visible effect in the editor, because we don't currently show any preview of the dismiss button in the block editor (though we could potentially add one if you feel it's warranted, @thomasguillot).

Closes #41.

### How to test the changes in this Pull Request:

1. Check out this branch and run `npm run build` to compile assets.
2. View an existing inline or overlay campaign on the front-end. Confirm that there's no difference in display compared to master.
3. Edit the existing campaign. Set the "Dismiss button alignment" option to left, center, and right and confirm that the button appears aligned as specified on the front-end.
4. Repeat with a new campaign.

### Other information:
